manifest/default.xml

```diff
@@ -22,7 +22,7 @@
 
   <!-- Build Scripts (required on CI servers) -->
   <project name="build" path="cbbuild" remote="couchbase">
-    <annotation name="VERSION" value="1.5.1"     keep="true"/>
+    <annotation name="VERSION" value="2.0.0"     keep="true"/>
```

Comment:
So with this change, I guess the 1.5.1 builds are going to switch from being commits from master (previously), to commits from 1.5.1 (branched off release/1.5.0).  Not a big problem or anything, just clarifying the expected behavior on the build machine

Reply:
Right.  It's not ideal (we'd originally planned to cut 1.5.1 from master), but need to move around the releases.